frontend/src/components/MoEntry/MoOrganisationUnitEntry.vue

```diff
@@ -13,6 +13,12 @@
         required
       />
 
+      <mo-input-text
+        :label="$t('input_fields.user_key')"
+        :placeholder="$t('input_fields.user_key_placeholder')"
+        v-model="entry.user_key"
```

Comment:
Should we make this a required field?
```suggestion
        v-model="entry.user_key"
        required
```

Reply:
Det ved jeg ikke. jeg tror vi bliver nødt til at spørge Alex - og du er jo tættest på ham for tiden? Med mindre du da har sendt ham på vej imod KBH.

Reply:
Jævnfør anden kommentar: Det er kun nogle gange, de vil angive den, så required tror jeg ikke er godt.